ToDoList/ToDoList/src/ToDoList.Api/ToDoController.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace ToDoList.src.ToDoList.Api
+{
+    public class ToDoController : Controller
+    {
+        public IActionResult Index()
+        {
+            return View();
+        }
+    }
```

Comment:
Empty `[ApiController]`, not MVC.
https://learn.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?view=aspnetcore-6.0&tabs=visual-studio